src/NzbDrone.Core/Notifications/Slack/SlackService.cs

```diff
@@ -30,18 +30,17 @@ public SlackService(Logger logger)
 
         public void OnDownload(DownloadMessage message, SlackSettings settings)
         {
-            var payload = new SlackPayload()
+            var payload = new SlackPayload
             {
                 IconEmoji = settings.Icon,
-                Username = settings.BotName,
+                Username = settings.Username,
                 Text = "Downloaded",
-                Attachments = new List<Attachment>()
+                Attachments = new List<Attachment>
                 {
                     new Attachment()
                     {
                         Fallback = message.Message,
                         Title = message.Series.Title,
-                        TitleLink = $"http://www.imdb.com/title/{message.Series.ImdbId}/",
                         Text = message.Message,
                         Color = "good"
```

Comment:
The color is "magic" that slack supports: https://api.slack.com/docs/attachments.